### PR TITLE
initial nits

### PR DESCRIPTION
**Pull request by chrismetz09**
Some initial suggestions. Use as appropriate.

### CODE REVIEW

File: content/posts/docker-prune.md [hunk 1]
```diff
@@ -9,12 +9,23 @@ ShowToc: true
 draft: true
 ---
 
-Docker a is platform for developing, shipping and running applications in isolated, lightweight, portable containers. It is a critical part of a developer's toolbelt. I use it everyday.
+[//]: # (elementary question but is there direct coorelation between imaze size and consumed disk?)
```
Comment (johnnymetz) [inline content/posts/docker-prune.md]: Yes, the larger the image, the more disk space it consumes.

We're not just talking about images though. There are many different Docker objects, such as images, containers, volumes, and networks.

File: content/posts/docker-prune.md [hunk 2]
```diff
@@ -30,6 +41,8 @@ The "Active" column represents objects that are associated with running containe
 
 Docker is eating up roughly 23.5 GB of storage capacity. 23 GB of that is reclaimable (or unused). Yikes! Let's free that up.
 
+[//]: # (what is a typical size? I heard about 5GB. Any idea based on your experience?)
```
Comment (johnnymetz) [inline content/posts/docker-prune.md]: Typical size of what?

I'm not talking about a single Docker image. All Docker objects combined on my laptop is taking up 23.5 GB or storage space.

File: content/posts/docker-prune.md [hunk 3]
```diff
@@ -9,12 +9,23 @@ ShowToc: true
...
+Docker is a platform for developing, shipping and running applications in isolated, lightweight and portable containers. It is a critical part of a developer's toolbelt and one I use just about everyday.
+
+Docker can consume a large amount of disk space. Per the [Docker documentation](https://docs.docker.com/config/pruning/):
 
 > Docker takes a conservative approach to cleaning up unused objects (often referred to as "garbage collection"), such as images, containers, volumes, and networks: these objects are generally not removed unless you explicitly ask Docker to do so. This can cause Docker to use extra disk space.
 
+The problems you might encounter with a large docker image include:
+
+* ...
+* ...
```
Comment (johnnymetz) [inline content/posts/docker-prune.md]: Great idea. I need to look up the negative effects of disk space bloat.

**Comment by johnnymetz** [comment]
I would love to update some things on my own but I'm not sure how to pull down your branch. I think you need to "Allow edits from maintainers": https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by chrismetz09** [comment]
I set "allow edits from maintainers" in PR. You need to set remote to chrismetz09/devUodates, right?

File: content/posts/docker-prune.md [hunk 4]
```diff
@@ -30,6 +39,8 @@ The "Active" column represents objects that are associated with running containe
 
 Docker is eating up roughly 23.5 GB of storage capacity. 23 GB of that is reclaimable (or unused). Yikes! Let's free that up.
 
+[//]: # 'what is a typical size? I heard about 5GB. Any idea based on your experience?'
```
Comment (johnnymetz) [inline content/posts/docker-prune.md]: Not sure. I'm talking about total disk usage in this article, not just images.

**Pull request merged**